### ISSUE TITLE
Default value for 'Use previous alignments' in Relion

### ISSUE THREAD

**Issue by delarosatrevin** [I2PC/scipion#1402]
From @JuhaHuiskonen...

I noticed that 'consider previous alignment' is a default in relion refinement protocols.   If this is selected, the current angles get written as priors, which is what it should do. But I don't think it makes sense to have this as a default however. 
 
I am saying this because I think it causes a problem in most typical workflows, where the angles come from 3D classification with a very coarse sampling. E.g. the angels can be off by let's say 7.5 degrees. In auto-refine, they get first refine in 'global mode'. But when the refinement switches to 'local mode', the local mode focuses on the Prior angle, not the best angle from the current refinement.  

In other words, having 'consider previous alignment' makes only sense, when the user wants to skip global alignment and switch directly to local alignment, for example in focused classification (or localised reconstruction).  

In summary, I would not have this as a default. And perhaps describe the behaviour (that the old angles are considered as Priors) in the help. 

@azazellochg , @josuegbl  Can you comment on this? I agree with Juha's point. 
@pconesa I think if we agree on the change, it should be very easy to implement (just put False as default value in that option), so it could enter in the release.

**Comment by pconesa** [comment]
Thanks, if all both agree, yes it can go in the release. By the way, I don't see it happening this year!.

**Comment by azazellochg** [comment]
Hi @delarosatrevin ! I agree with @JuhaHuiskonen , the current default option will set priors and thus all subsequent local searches are done around those. Switching this off will allow optimal angles/shifts (from previous iteration) to slowly move away.

However, now the alignments are not copied from input particles at all to input star file if this options is False. If someone whats to continue from previous iteration (but not stick to priors), continue option should be used with copyAlignment=False. But what if I want to do 3D classification using angles from 3D refinement without sticking to priors? Continue cannot be used now, because it is a different protocol.

**Comment by pconesa** [comment]
Thanks @azazellochg . @JuhaHuiskonen , @delarosatrevin ... Is this becoming a not as straightforward as initially thought?

**Comment by JuhaHuiskonen** [comment]
Hi @azazellochg @pconesa @delarosatrevin so should there by two separate options:
1. consider previous alignment (not a default - if selected will copy the alignments but not the priors)
2. consider alignment as priors (not a default and perhaps this should be an advanced option evoked only if 1 is chosen - if selected will copy the alignments also as priors).

**Comment by josuegbl** [comment]
When running refine/classif. in continue mode Relion uses  --continue parameter, refers to the optimiser file that points to the data file of previous run. In other words, an input_particles.star is created but not used by Relion. Maybe is better not generate the input_particles.star. I agree with set to False as default value to 'consider previous alignment' param.

**Comment by pconesa** [comment]
Let's do it then!
